docs/gameplay-features/spawners.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Information on obtaining and using spawners in Theatria.
+---
+
+# Spawners
+
+Spawners are a valuable resource in Theatria, allowing players to generate specific mobs. Here's how you can obtain and use them effectively.
+
+## How to Obtain Spawners
+
+1. **Ranks**
+   Unlock spawners through [ranks](ranks/ethereal-ranks), which can be purchased with in-game currency. Higher ranks often include better rewards.
```

Comment:
The first kit for spawners happens at hero rank:
https://docs.playtheatria.com/ranks/mythic-tier/hero